app/utils.py

```diff
@@ -106,29 +106,41 @@ def standardize_response(payload={}, status_code=200):
     err_map = {
         400: "Bad Request",
         401: "Unauthorized",
-        403: "Unauthorized",
+        403: "Forbidden",
         404: "Not Found",
         405: "Method Not Allowed",
         422: "Unprocessable Entity",
         429: "Rate Limit Exceeded",
         500: "Server Error"
     }
 
+    msg_map = {
+        400: "Bad Request. Did you provide valid JSON?",
+        401: "You must provide a valid API token in the x-apikey header.",
+        403: "This endpoint is forbidden.",
+        404: "The resource you requested does not exist.",
+        405: "This method is not allowed.",
+        422: "This request failed validation",
+        429: "You have exceeded your rate limit. Try again later.",
+        500: "Something went wrong"
+    }
+
     if status_code >= 400 and err_map.get(status_code):
         resp["status"] = err_map.get(status_code)
         resp["status_code"] = status_code
 
         if errors:
             resp["errors"] = errors
         else:
-            resp["errors"] = [{
-                "code": '-'.join(err_map.get(status_code).split(' ')).lower()
-            }]
+            code = '-'.join(err_map.get(status_code).split(' ')).lower()
+            message = msg_map.get(status_code)
+            resp["errors"] = {}
+            resp["errors"][code] = {"message": message}
 
     elif not data:
-        resp["errors"] = [{"code": "something-went-wrong"}]
+        resp["errors"] = {'errors': {"server-error":{"message":"Something went wrong"}}}
```

Comment:
TODO: we need coverage around this code

Reply:
This one is kind of weird... I think I'll have a test that mocks the standarized response itselt and just returns standardized_response(1337) which will fail the if statement.